.github/workflows/o3_schedule.yaml

```diff
@@ -6,7 +6,21 @@ on:
   schedule:
     - cron:  '0 6 * * MON'
 jobs:
+  check:
```

Comment:
I'd rather call it `latest_version` or something like that.
But this is minor suggestion of course :)

Reply:
done!